content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -292,82 +351,75 @@ Before `spec.data` gets changed from `atomic` to `granular`,
 without a conflict), but it no longer owns `key1` and `key2`, so another
 manager can then modify or delete those fields without conflict.
 
-## Custom Resources
-
-By default, Server Side Apply treats custom resources as unstructured data. All
-keys are treated the same as struct fields, and all lists are considered atomic.
-
-If the Custom Resource Definition defines a
-[schema](/docs/reference/generated/kubernetes-api/{{< param "version" >}}#jsonschemaprops-v1-apiextensions-k8s-io)
-that contains annotations as defined in the previous "Merge Strategy"
-section, these annotations will be used when merging objects of this
-type.
 
 ## Using Server-Side Apply in a controller
 
-As a developer of a controller, you can use server-side apply as a way to
+As a developer of a controller, you can use Server-Side Apply as a way to
 simplify the update logic of your controller. The main differences with a
 read-modify-write and/or patch are the following:
 
 * the applied object must contain all the fields that the controller cares about.
 * there is no way to remove fields that haven't been applied by the controller
-  before (controller can still send a PATCH/UPDATE for these use-cases).
-* the object doesn't have to be read beforehand, `resourceVersion` doesn't have
+  before (controller can still send a **patch** or **update** for these use-cases).
+* the object doesn't have to be read beforehand; `resourceVersion` doesn't have
   to be specified.
 
-It is strongly recommended for controllers to always "force" conflicts, since they
-might not be able to resolve or act on these conflicts.
+It is strongly recommended for controllers to always force conflicts on objects that
+the own and manage, since they might not be able to resolve or act on these conflicts.
 
-## Transferring Ownership
+## Transferring ownership
 
 In addition to the concurrency controls provided by [conflict resolution](#conflicts),
-Server Side Apply provides ways to perform coordinated
+Server-Side Apply provides ways to perform coordinated
```

Comment:
The feature is called "Server Side Apply"?

Reply:
Tech reviewers: what is the right name?

Reply:
It's fairly inconsistent, but I would say that `Server-Side Apply` wins.

```sh
apelisse ~/code/website> rg -i "server side apply" -c
content/zh/docs/reference/using-api/api-concepts.md:4
content/zh/docs/reference/using-api/server-side-apply.md:18
content/en/docs/reference/using-api/api-concepts.md:4
content/en/docs/reference/using-api/server-side-apply.md:18
apelisse ~/code/website> rg -i "server-side apply" -c
content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:1
content/en/blog/_posts/2020-04-01-server-side-apply-beta2.md:7
content/en/blog/_posts/2018-07-16-kubernetes-1-11-release-interview.md:1
content/en/blog/_posts/2019-01-14-apiserver-dry-run-and-kubectl-diff.md:2
content/en/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:1
content/en/blog/_posts/2019-06-20-Future-of-CRDs-Structural-Schemas.md:1
content/ko/docs/setup/release/notes.md:2
content/zh/docs/tasks/manage-kubernetes-objects/declarative-config.md:1
content/zh/docs/setup/release/notes.md:1
content/zh/docs/reference/using-api/server-side-apply.md:15
content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md:1
content/en/docs/reference/using-api/server-side-apply.md:13
static/docs/reference/generated/kubectl/kubectl-commands.html:2
```